### PR TITLE
Update the PulumiUP OpenGraph image

### PR DESCRIPTION
**Pull request by cnunciato**
Fixes https://github.com/pulumi/marketing/issues/116.

### CODE REVIEW

**Comment by cnunciato** [comment]
@SaraDPH In a few minutes this PR will generate a preview that we should be able to use to verify that the image looks like what we expect.

File: themes/default/content/pulumi-up/_index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: PulumiUP
 type: page
 layout: pulumi-up-2022
-meta_image: /images/pulumiup/save-the-date-pulumiup.png
+meta_image: /images/pulumiup/2022-save-the-date.png
 meta_desc: |
     PulumiUP introduces you to the emerging practice of Cloud Engineering. Hear from the Pulumi team
     and assorted industry experts.
```
Comment (cnunciato) [inline themes/default/content/pulumi-up/_index.md:8]:
```suggestion
meta_image: /images/pulumiup/2022-save-the-date.png
meta_desc: |
    DIFFERENT WORDS
```

**Comment by cnunciato** [comment]
@SaraDPH Here's how the Twitter and LinkedIn previews look:

image

image

Did you want to make any changes to the text copy as well? If so, you can make a suggestion directly in the GitHub UI by clicking the Files Changed tab (☝️) and then dragging the plus icon and clicking the suggestion button:

https://user-images.githubusercontent.com/274700/158703604-5e5b2bfb-52eb-4612-9400-1b0416f1e74c.mov

**Comment by SaraDPH** [comment]
@cnunciato the suggestion button doesn't appear to me... can I just write _'''suggestion_ or it may break something?!

image

**Comment by susanev** [comment]
Chatted with Sara, this was because she was looking a code file outside of the context of the pr, once she went directly to the files changed tab of the pr she was able to interact with the suggested changes button.

File: themes/default/content/pulumi-up/_index.md (hunk 1 above)
Comment (SaraDPH) [inline themes/default/content/pulumi-up/_index.md:8]:
```suggestion
meta_desc: |
    Join us for PulumiUP 2022: free virtual conference on May 4th. Let's talk about the future of IaC, Cloud Engineering & #DevOps and Cloud. Hear from industry-recognized leaders, attend technical talks, demos, and panel discussions. #PulumiUP #infrastructureascode
```

File: themes/default/content/pulumi-up/_index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: PulumiUP
```
Comment (SaraDPH) [inline themes/default/content/pulumi-up/_index.md:2]:
```suggestion
title: PulumiUP: Virtual Cloud Engineering Event | May 4-5 2022
```

File: themes/default/content/pulumi-up/_index.md (hunk 1 above)
Comment (zchase) [reply themes/default/content/pulumi-up/_index.md:8]: @SaraDPH we have some validation that only allows meta descriptions to be at most 160 characters. We don't currently support separate text for social cards, but it is something we can implement!

Comment (cnunciato) [reply themes/default/content/pulumi-up/_index.md]: Definitely support adding this! Would be good to submit it as a follow-up PR, though -- I'll make an issue for that and add it to the list. (Should be a quick follow-up to this one.)

**Comment by SaraDPH** [comment]
The description looks good.
Did you see my suggestion for the Title? Please update as that is what it shows as a headline on search

image

**Comment by cnunciato** [comment]
@SaraDPH Yes! I committed your suggestion (thanks!), but it looks like it's got a minor syntax error. I'll fix that up and then we should be good to go.

**Review by SaraDPH — APPROVED** [review]

**Pull request merged**